src/XRP/shared/gateway-balances.ts

```diff
@@ -0,0 +1,80 @@
+import { XrpUtils } from 'xpring-common-js'
+import { XrpError, XrpErrorType } from '.'
+import {
+  GatewayBalancesResponse,
+  CurrencyValuePair,
+} from './rippled-json-rpc-schema'
+
+/**
+ * Represents an amount of an Issued Currency.
+ */
+export class IssuedCurrencyValue {
```

Comment:
Don't you want the issuer to be stored here as well?

Reply:
Nope... this type appears in the `gateway_balances` response where the issuer is always understood to be the account provided in the `gateway_balances` query, so it's different from the protobuf that represents issued currencies.